crates/configuration/src/shared/types.rs

```diff
@@ -406,6 +406,7 @@ impl<'de> Deserialize<'de> for AssetLocation {
 pub enum Arg {
     Flag(String),
     Option(String, String),
+    Array(String, Vec<String>),
```

Comment:
In theory we could refactor `Option` to support `Vec<String>` instead of `String`.
I thought it would be more clear to distinguish multi-value arguments from the single-value ones.
I don't have strong opinion here, I can rework if it seems better to use `Option` only :)